CMakeLists.txt

```diff
@@ -1862,7 +1862,9 @@ if (TOOL_DR_MEMORY AND WIN32)
   add_subdirectory(drstrace)
 endif (TOOL_DR_MEMORY AND WIN32)
 
-add_subdirectory(drltrace)
+if (NOT APPLE)
```

Comment:
This needs a FIXME i#xxxx comment so it's clear this is temporary and there's something to be fixed.